### PR TITLE
Portfolio Website

### PR DESCRIPTION
**Pull request by justinrichey**
This is a pull request for the portfolio website (week 1). These changes include adding about me, project, and link webpages. The website just contains text, links, and one image. The main changes made were to the style, where I centered the title/navbar and made a dark theme. The JavaScript file is a single function that loads pages based on what the user clicked on the navbar.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -12,17 +12,6 @@
...
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+function displayText(pageType) {
+    window.location = pageType + ".html";
+}
```
Comment (dcollier) [inline portfolio/src/main/webapp/script.js]: can you leave a newline at the end of the file?

Comment (justinrichey) [reply portfolio/src/main/webapp/script.js]: yup, should be fixed now 👍

**Comment by justinrichey** [comment]
Hey everyone!  I accidentally forgot to merge my last pull request. Since then, I've committed for step 1 of week 2, and my commits are included as apart of this pull request. All I did during those commits was pull to the virtual machine and change "Hello World" to "Hello Team" in DataServlet.java. Does everything look good to merge?

**Comment by jtajonera** [comment]
This looks all good to me!

**Comment by justinrichey** [comment]
Great, thanks!

**Pull request merged**